crates/bench/.gitignore

```diff
@@ -0,0 +1,4 @@
+callgrind.*.out
+*_expanded.rs
```

Comment:
Can you comment on what creates these, and why they can't go in eg `target/` somewhere?

Reply:
They were getting generated when I was experimenting manually, no longer. I'll purge